RaveBusinessLogic/TauDEM.xml

```diff
@@ -9,58 +9,48 @@
                 <Children collapsed="true">
                   <Node label="DEM" xpath="Raster[@id='DEM']" type="raster" symbology="dem" transparency="40" id="dem" />
                   <Node label="Channel Area" xpath="Geopackage/Layers/Vector[@id='CHANNEL_AREA']" type="polygon" id="channel_area" symbology="ChannelArea" />
-                  <Node label="Drainage Input(s)">
-                    <Children>
-                        <Node label="Channel Area" xpath="Geopackage/Layers/Vector[@id='CHANNEL_AREA']" type="polygon" id="channel_area" symbology="ChannelArea" />
-                        <Node label="Channel Line Network" xpath="" type="line" id ="" symbology="" />
-                    </Children>
-                    </Node>
```

Comment:
This should come back if we can get it to work... Just comment out for now.  Also line 15 needs actual path and symbology.